.editorconfig

```diff
@@ -1,8 +1,6 @@
 root = true
 
 [*]
-end_of_line = lf
```

Comment:
Reason for this change?

Reply:
This is what it looks like on Windows system, and it's just annoying to edit source code like this.

![2024-03-08 15 01 16](https://github.com/editorconfig/editorconfig-emacs/assets/8685505/aa1a918c-fe95-4232-a754-c30d2c53cc55)

Reply:
It feels like this is related to your git settings. Have you tried turning off `git` [autocrlf](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreautocrlf)?

```
git config --global core.autocrlf false
```

Reply:
All the projects I've cloned don't have this issue. I prefer to change the setting here. 🤔

Reply:
If we remove this option, someone might comes in and mix crlf and lf. Because EditorConfig already transforms the EOLs here, `git` `autocrlf` settings would mess up because it transforms `lf` to `crlf`.

Reply:
How about remove this second commit for now and we can discuss in a separate PR?

Reply:
> If we remove this setting, someone might comes in and mix crlf and lf. Because EditorConfig already transforms the EOLs here, git autocrlf settings would mess up because it transforms lf to crlf.

Why not configure this in gitattributes? 🤔 

> How about remove this second commit for now and we can discuss in a separate PR?

Sure.